packages/mgt-chat/src/components/ChatListItem/ChatListItem.tsx

```diff
@@ -128,22 +127,17 @@ export const ChatListItem = ({ chat, myId, isSelected, isRead }: IMgtChatListIte
 
   // manage the internal state of the chat
   const [chatInternal, setChatInternal] = useState(chat);
-  const [read, setRead] = useState<boolean>(isRead);
-  const cache = new LastReadCache();
 
   // when isSelected changes to true, setRead to true
   useEffect(() => {
     if (isSelected) {
-      setRead(true);
+      isRead = true;
```

Comment:
Changing the value of a property like this does not force a re-render, so I don't suspect this is doing anything. You already have a check in ChatList to cover isSelected. I suspect you can just delete this useEffect entirely.